imap_processing/tests/ccsds/test_excel_to_xtce.py

```diff
@@ -12,21 +12,36 @@
 
 
 @pytest.fixture()
-def filepath(tmpdir):
-    p = Path(tmpdir / "test_file.xlsx").resolve()
-    p.touch()
+def excel_file():
+    p = Path(__file__).parent / "test_data" / "excel_to_xtce_test_file.xlsx"
     return p
 
 
+def test_generated_xml(excel_file, tmp_path):
+    """Make sure we are producing the expected contents within the XML file.
+
+    To produce a new expected output file the following command can be used.
```

Comment:
Good to know! This makes me wonder, do we have readme for this cli tool?

Reply:
We have an issue to make this better! On my list...
https://github.com/IMAP-Science-Operations-Center/imap_processing/issues/703